README.md

```diff
@@ -44,3 +44,8 @@ services:
 $logProcessor = $this->container->get('Keboola\\ErrorControl\\Monolog\\LogProcessor');
 $logProcessor->setLogInfo(new LogInfo(...));
 ```
+
+## Development
+Use the provided `test-cf-stack.json` to create a CloudFormation stack. Use the outputs to set environment variables
```

Comment:
Součástí outputů CF je jenom region a ne S3:
![only-region](https://user-images.githubusercontent.com/5272798/60094326-07761a80-974c-11e9-8c47-26fb41b03beb.png)

Je to prkotina, ale asi bych upravil tu větu nebo to doplnil od CF.

Reply:
fixed